artifactory/usage/reportusage.go

```diff
@@ -67,8 +71,16 @@ func isVersionCompatible(artifactoryVersion string) bool {
 	return version.AtLeast(minArtifactoryVersion)
 }
 
-func reportUsageToJson(productId, commandName string) ([]byte, error) {
+func reportUsageToJson(productId, commandName string, attributes ...ReportUsageAttribute) ([]byte, error) {
 	featureInfo := feature{FeatureId: commandName}
+	if len(attributes) > 0 {
+		featureInfo.Attributes = make(map[string]string, 0)
```

Comment:
It is a good practice to set the capacity, if possible:
```suggestion
		featureInfo.Attributes = make(map[string]string, len(attributes))
```

Reply:
done